### DISCUSSION TITLE
7/26(수) tuist edit을 했을 때 수정하는 방법을 알아와주세요

### DISCUSSION THREAD

**Discussion by jjunhaa0211** [Info-iOS/iOStudy#199] [General]
먹젠을 하다보면 지금 이상태로 두면 박준하 이자식 모듈 잘못짠 것 같은데? 이런 상황이 있을 수 있기 때문에 모듈을 보는 방법을 알고 서로간의 의존관계를 파악하는 것이 좋을 것 입니다. 물론 방학 끝나면 저도 tuist를 따로 공부중이라 알려드리겠습니다. 일단 프젝 때문에 예습하는 구나~ 생각하지고 제일 기초가 되면 tuist edit을 알아와주세요 🍀

**Comment by Eunho0922** [comment]
넹
